src/material/expansion/_expansion-theme.scss

```diff
@@ -17,7 +17,7 @@
     border-top-color: mat-color($foreground, divider);
   }
 
-  .mat-expansion-panel:not(.mat-expanded) .mat-expansion-panel-header {
+  .mat-expansion-panel .mat-expansion-panel-header {
```

Comment:
I think the main purpose of the `:not` was to prevent the hover indication from showing while open. Maybe we should move it there rather than removing it completely? Otherwise it looks like this:

https://i.imgur.com/J3Y5x5t.png

Reply:
Move it where?

Reply:
To the focus selectors below so this would become:

```scss
.mat-expansion-panel {
  & .mat-expansion-panel-header.cdk-keyboard-focused,
  & .mat-expansion-panel-header.cdk-program-focused,
  &:not(.mat-expanded) .mat-expansion-panel-header:hover {
    &:not([aria-disabled='true']) {
      // ...
    }
  }
}
```

Reply:
Coming back to this, I realize I misread your original comment as "focus" instead of "hover" and was confused. Made the suggested change.